### PR TITLE
APR improved config

### PR DESCRIPTION
**Pull request by gianipez**

### CODE REVIEW

File: test/triggerTest.fcl [hunk 1]
```diff
@@ -24,11 +24,10 @@ services.TFileService.fileName : "/dev/null"
 
 # timing information
 services.TimeTracker : {
-   dbOutput : {
-      filename : ""
-      #	filename : "runGlobalTrigger.db" #uncomment to create the .db output
-      overwrite : true
-   }
+  dbOutput : {
+    filename : "runGlobalTrigger.db" #uncomment to create the .db output
```
Comment (brownd1978) [inline test/triggerTest.fcl:28]: I think creating the database should be disabled by default

**Comment by gianipez** [comment]
This is ready to go. @kutschke this PR will change the trigger counts in the nightly. Please, let me know if it's ok to merge it today.

**Review by kutschke — COMMENTED** [review]
As I read this PR.  This addition of the filters with module_type: @nil and the definition of 3 timing paths are preparation for future work.  They are not runable now because the @nil has not been overridden.   Is that right?

**Comment by kutschke** [comment]
> This is ready to go. @kutschke this PR will change the trigger counts in the nightly. Please, let me know if it's ok to merge it today.

I just added one question and @brownd1978 still needs to sign off on the changes he requested.  After that I will merge.

**Comment by gianipez** [comment]
Ho @kutschke , I think you got a bit confused.
- the place holders are needed only because if you don’t enable all the defined sequences when generating the menu files, no PS event filter will be defined and art will complain of a missing module instance declaration
- the timing sequences are not part of the physMenu as I include them only during the timing tests
- The trigger counts will change only in the apr trigger paths defined in the physMenu as enabled 

I integrated Dave’s request so this can be merged.

**Review by brownd1978 — APPROVED** [review]

**Comment by kutschke** [comment]
> * the place holders are needed only because if you don’t enable all the defined sequences when generating the menu files, no PS event filter will be defined and art will complain of a missing module instance declaration

Thanks Giani.  Ok - I think I understand.  The code that needs them defined is the menu generator which does not care how they are defined, just that they are define.  Is that right? 

I will merge now but I would still like to be sure that I understand.   So let me know if I understood correctly.

**Comment by gianipez** [comment]
> > * the place holders are needed only because if you don’t enable all the defined sequences when generating the menu files, no PS event filter will be defined and art will complain of a missing module instance declaration
> 
> Thanks Giani. Ok - I think I understand. The code that needs them defined is the menu generator which does not care how they are defined, just that they are define. Is that right?
> 
> I will merge now but I would still like to be sure that I understand. So let me know if I understood correctly.

It's not the python script that wants them defined, it's `art`, because in the final #include cascade, we include the file that houses all the sequences where those filters are part of the sequences no matter If they are enabled or not.

**Pull request merged**